src/main/resources/db/migration/V5__AddedPseudonymToPublicAccount.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE public_account
```

Comment:
Let's make version name to start with V<current_version>.0.<branch_name>
e. g. V5.0.37__

It will reduce feature merging conflicts.